extensions/ql-vscode/src/query-results.ts

```diff
@@ -89,9 +92,8 @@ export class CompletedQuery implements QueryWithResults {
   }
 
   getLabel(): string {
-    if (this.options.label !== undefined)
-      return this.options.label;
-    return this.config.format;
+      return this.options?.label
+        || this.config.format;
```

Comment:
Is changing behavior on `this.options?.label == ""` desirable? (before it would have returned it, now it falls through to `this.config.format`) Kind of a corner case, and maybe your change is for the better; if the user explicitly tries to set the label to empty string, maybe we should behave as if they want the default back. Just wanted to make sure we're both aware.

Reply:
Yes. That's what I was thinking. I don't think people would want an empty label.